packages/fhi-designsystem/src/components/fhi-text-input/fhi-text-input.stories.ts

```diff
@@ -0,0 +1,171 @@
+      ?readonly=${args.readonly}
+      ?disabled=${args.disabled}
+    ></fhi-text-input>`,
+  argTypes: {
+    name: {
+      control: 'text',
+      description:
+        'Setter navn på inputfeltet. Dette brukes typisk av assosierte form-elementer for å identifisere feltet i FormData.',
+      defaultValue: { summary: 'undefined' },
+    },
+    value: {
+      control: 'text',
+      description:
+        'Setter verdi på inputfeltet. Dette vil være synlig i feltet og automatisk bli med i FormData.',
+      defaultValue: { summary: '' },
+    },
+    label: {
+      control: 'text',
+      description:
+        'Setter label. Dette assosieres med inputfelted og vises over. Om dette er satt trenger du ikke å deklarere eget label-element.',
+      defaultValue: { summary: 'undefined' },
+    },
+    message: {
+      control: 'text',
+      description:
+        'Setter melding som vises til brukeren. Dette vises under inputfeltet og er ment for å gi veiledning til brukeren.',
+      defaultValue: { summary: 'undefined' },
+    },
+    placeholder: {
+      control: 'text',
+      description:
+        'Setter placeholder for inputfeltet. Denne vil ikke være synlig om feltet har verdi.',
+      defaultValue: { summary: 'undefined' },
+    },
+    readonly: {
+      control: 'boolean',
+      description: 'Bestemmer om feltet kan redigeres.',
+      defaultValue: { summary: false },
+    },
+    disabled: {
+      control: 'boolean',
+      description: 'Bestemmer om bruker kan samhandle med feltet.',
+      defaultValue: { summary: false },
+    },
+    status: {
+      control: 'select',
+      description:
+        'Bestemmer om feltet har en status. Dette vil endre utseende på feltet.',
+      defaultValue: { summary: 'undefined' },
+      options: ['error'],
+    },
+  },
+};
+
+type Story = StoryObj<FhiTextInput>;
+
+export const Preview: Story = {
+  tags: ['!dev'],
+  args: {
+    label: 'Label',
+    message: 'Message',
+  },
+};
+
+export const Error: Story = {
+  tags: ['!dev'],
+  args: {
+    label: 'Label',
+    message: 'Informative error message',
+    status: 'error',
+    value: 'This field has an error',
+  },
+};
+
+export const Disabled: Story = {
+  tags: ['!dev'],
+  args: {
+    label: 'Label',
+    value: 'This field is disabled',
+    disabled: true,
+  },
+};
+
+export const Readonly: Story = {
+  tags: ['!dev'],
+  args: {
+    label: 'Navn',
+    value: 'Ola Nordmann',
+    readonly: true,
+  },
+};
+
+export const WithNoAttributes: Story = {
+  name: 'Default',
+  args: {},
+};
+
+export const WithLabelMessagePlaceholder: Story = {
+  name: 'Label and message',
+  args: {
+    label: 'Adresse',
+    message: 'Vennligst skriv inn bostedsadresse',
+  },
+};
+
+export const WithLabelMessageValueError: Story = {
+  name: 'Error',
+  args: {
+    label: 'E-postadresse',
+    message:
+      'Ufullstendig adresse, sjekk at alfakrøll (@) og domene er med og riktig skrevet (f.eks: designsystemet@fhi.no)',
```

Comment:
```suggestion
      'Ufullstendig adresse, sjekk at krøllalfa (@) og domene er med og riktig skrevet (f.eks: designsystemet@fhi.no)',
```

I dag lærte jeg at det heter "krøllalfa" og ikke "alfakrøll".